config/initializers/keybase.rb

```diff
@@ -0,0 +1,9 @@
+module Keybase
+  def self.local_domain
+    'mastodon.social'
```

Comment:
i'm not really sure what the right way is to do this. maybe a new environment variable or piggy backing on the `LOCAL_DOMAIN` configuration. tbd.